database/index.js

```diff
@@ -0,0 +1,37 @@
+require('dotenv').load();
+var Sequelize = require('sequelize');
+
+var user = process.env.PG_USER;
+var password = process.env.PG_PASSWORD;
+var database = process.env.PG_DATABASE;
+var host = process.env.PG_HOST;
+var port = process.env.PG_PORT;
+
+var db = (module.exports = new Sequelize(database, user, password, {
+  host: host,
+  port: port,
+  dialect: 'postgres',
+  logging: false,
+  pool: {
+    max: 5,
+    min: 0,
+    idle: 10000
+  },
+  operatorsAliases: false
+}));
+
+var Project = (module.exports.Project = require('./project')(db));
+var Item = (module.exports.Item = require('./item')(db));
+var Comment = (module.exports.Comment = require('./comment')(db));
+var Tag = (module.exports.Tag = require('./tag')(db));
+var Stat = (module.exports.Stat = require('./stat')(db));
+
+Item.belongsTo(Project, { foreignKey: 'project_id' });
+Tag.belongsTo(Project, { foreignKey: 'project_id' });
```

Comment:
Shouldn't tags belong to Item and not Project?

Reply:
Per discussion with @batpad, we're going to keep this as-is since we will want to fetch all tags at the project-level.